upload/src/addons/SV/WarningImprovements/XF/Admin/Controller/Warning.php

```diff
@@ -41,12 +36,20 @@ public function actionIndex(ParameterBag $params)
                 unset($actions['']);
             }
 
+            $globalWarnings = [];
+            if (!empty($warnings['']))
+            {
+                $globalWarnings = $warnings[''];
+                unset($warnings['']);
+            }
+
             $escalatingDefaults = $this->finder('SV\WarningImprovements:WarningDefault')->fetch();
 
             $response->setParams([
                 'categoryTree' => $categoryTree,
 
                 'warnings' => $warnings,
+                'globalWarnings' => $globalWarnings,
```

Comment:
A warning should always belong to a category (organizationally it just makes things sane, as the warning category is used as a label)

Reply:
Probably should state these need to be moved into a category so user's can fix things up. But they should not be able to remove the category